loops/break.py

```diff
@@ -0,0 +1,11 @@
+print "*******break and continue******"
+iter = 1
```

Comment:
iter is not required, Any way it is going to infinite loop. Better use
`while True:
            statments
`